### PR TITLE
Motor Fault Detection at the Edge Documentation

### PR DESCRIPTION
**Pull request by seldauyanik-maxim**

### CODE REVIEW

**Review by rotx-maxim — DISMISSED** [review]

File: Guides/MAX78000 Motor Monitoring Case Study with SampleMotorDataLimerick Dataset.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Details for other fault types:
+<img src="resources/SampleMotorDataLimerick_otherFaults.png" alt="Other Faults" style="zoom: 33%;" />
+
+
+
+Each fault was tested at 600, 1200, 1800, 2400 & 3000 RPM and tested both with and without an additional 11 lb (5 kg) load on the shaft. ADXL356 sensor data is used for vibration raw data.
+
+For ADXL356 sensor, the sampling frequency was **20 kHz** and data csv files recorded for **2 seconds** in the X, Y and Z directions.
+
+Data files are saved in `.csv` format with the following naming convention:
```
Comment (aniktash) [inline Guides/MAX78000 Motor Monitoring Case Study with SampleMotorDataLimerick Dataset.md:57]: I'd suggest to add an example of a csv file format here.

File: Guides/resources/SampleMotorDataLimerick_auto_encoder_I_O.png
Comment (vicloginov) [inline Guides/resources/SampleMotorDataLimerick_auto_encoder_I_O.png:1]: Is a first layer Conv2D or Conv1D?

File: Guides/MAX78000 Motor Monitoring Case Study with SampleMotorDataLimerick Dataset.md (hunk 1 above)
Comment (seldauyanik-maxim) [reply Guides/MAX78000 Motor Monitoring Case Study with SampleMotorDataLimerick Dataset.md:57]: * Sure, I think we can also link an actual csv file in our Datasets folder after it is published in GitHub
* For Limerick dataset it is a little more complicated: healthy identifier is the string '_GoB_GS_BaLo_WA_' in the file name wrt the file naming convention, reflecting a:  Good Bearing, Good Shaft, Balanced Load and Well Aligned

File: Guides/resources/SampleMotorDataLimerick_auto_encoder_I_O.png
Comment (seldauyanik-maxim) [reply Guides/resources/SampleMotorDataLimerick_auto_encoder_I_O.png:1]: True, implementation is Conv1D. Image also reflects Conv1D with 256 channels. Seems like a typo in the image title. I will update in the next commit. Thanks a lot

File: Guides/resources/SampleMotorDataLimerickl_auto_encoder_overall_arch.png
Comment (vicloginov) [inline Guides/resources/SampleMotorDataLimerickl_auto_encoder_overall_arch.png:1]: Typo in "Reconstructred"

Comment (seldauyanik-maxim) [reply Guides/resources/SampleMotorDataLimerickl_auto_encoder_overall_arch.png:1]: Thanks a lot, updated

**Review by vicloginov — APPROVED** [review]

**Review by rotx-maxim — APPROVED** [review]

**Review by MaximGorkem — APPROVED** [review]

**Pull request merged**